### PR TITLE
Move to NinesStack

### PR DESCRIPTION
**Pull request by relistan**
This updates all the files and deps to point to the NinesStack versions instead of Nitro. A little more work will have to go into the README, but this takes care of most of it and lets the project build and test properly. I will separately add this to Travis.

### CODE REVIEW

**Review by mihaitodor — APPROVED** [review]
LGTM, but you might want to ask David to also transfer https://github.com/Nitro/superside (and https://github.com/Nitro/traefik, although that's very much out of date now).

**Comment by relistan** [comment]
Thanks @mihaitodor I am ready to walk away from those. I will remove from the README.

**Comment by mihaitodor** [comment]
Sounds good to me. :shipit:

**Comment by mihaitodor** [comment]
Hey @jopecko, nice to see you around! Yeah, that can be annoying, but the new Go module system is more robust in this regard. I haven’t hit any casing issues with imports in a long time and we do have camel case Go repos at work. Not sure if @relistan still has issues with this.

**Comment by relistan** [comment]
Hey @jopecko ! Good to hear from you. Yeah, there was a big old problem some time ago when a popular Go library `logrus` was in an org that was renamed from caps to lower case. This was a huge problem for people on macos because you were physically not able to have both repos on your machine inside the `GOPATH`. Had he not renamed it, or had macos supported a case-sensitive filesystem this would have been fine. Alas, macos is still dumb in this regard:
```
$ touch asdf
$ mkdir Asdf
mkdir: Asdf: File exists
```
But that's OK because I'm not gonna rename the org! Haha

**Comment by sbz** [comment]
Hey folks 👋 ! 

@relistan You also might want to remove the Microbagder links from the README in the future PRs. They're planning next month to shut it down cf. [here](https://microbadger.com/shutdown). 

Could be a good reason to prepare new docker repository and use GitHub [packages](https://github.com/features/packages) to host the Docker images.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 Sidecar ![Sidecar](views/static/Sidecar.png)
 =====
 
-[![](https://travis-ci.com/Nitro/sidecar.svg?branch=master)](https://travis-ci.com/Nitro/sidecar)
+[![](https://travis-ci.com/NinesStack/sidecar.svg?branch=master)](https://travis-ci.com/NinesStack/sidecar)
 [![](https://images.microbadger.com/badges/image/gonitro/sidecar.svg)](https://microbadger.com/images/gonitro/sidecar)
```
Comment (relistan) [inline README.md:5]:
```suggestion
```

Comment (relistan) [inline README.md:5]:
```suggestion
```

**Comment by relistan** [comment]
Thanks @sbz !

**Pull request merged**